mon-pix/translations/en.json

```diff
@@ -722,7 +722,9 @@
             "modify": "Change student number",
             "not-existing": "I don't have a student number"
           }
-        }
+        },
+        "message": "This is the first time you have accepted the invitation from {organizationName}.\nPlease fill in these 4 fields as accurately as possible: Pix will then be able to find you in the student database and link your Pix account to it once and for all.",
+        "title": "{organizationName} has invited you to use Pix as part of your training."
```

Comment:
c'est bon on a la traduction en anglais :
`"Register your Pix account with <Univ>.

Enter the information as it appears on your student card to access the test and submit the results. Pix will remember it next time."`